awscli/topics/config-vars.rst

```diff
@@ -0,0 +1,209 @@
+:title: AWS CLI Configuration Variables
+:description: Configuration Variables for the AWS CLI
+:category: General
```

Comment:
I don't have a strong opinion either way.  If I had to pick, I'd go with "General", mainly because none of the other categories have "Topics" appended to them, and this is the topic guide list so it's implied these are all categories for topics.